Code/src/main/java/nl/utwente/group10/haskell/type/ConstT.java

```diff
@@ -1,6 +1,7 @@
 package nl.utwente.group10.haskell.type;
 
 import com.google.common.base.Joiner;
+import com.sun.org.apache.xpath.internal.functions.FuncFalse;
```

Comment:
Not at all, strange that it appears (because I have never used it). Downside of IDE magic I guess.